docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+  and continuous measures of skill.
+
+Additional verification and diagnostic approaches that can be helpful
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Geographical methods demonstrate where the error occurs geographically.
+
+  * METplus methods: Series-Analysis tool.
+    
+  * METplus line types: Most Grid-Stat and Point-Stat line types.
+    
+* Object Based measures the location error of the forecast and how the
+  total error break down into variety of descriptive attributes.
+   
+  * METplus methods: MODE, MTD, MvMODE, Grid-Stat Distance Maps.
+    
+  * METplus line types: MODE object attribute files, MODE CTS, MTD object
+    attribute files, MTD CTS, Grid-Stat DMAP.
+    
+* Neighborhood relaxes the requirement for an exact match by evaluating
+  forecasts in the local neighborhood of the observations.
+   
+  * METplus methods: Grid-Stat Neighborhood, Point-Stat HiRA, Ensemble-Stat
+    HiRA.
+
+  * METplus line types: NBRCTC, NBRCTS, NBRCNT, ECNT, ORANK, RPS.
+     
+* Domain Decomposition and Transform applies a transform to a given field
+  to identify errors on different spatial scales:
+   
+  * METplus methods: Grid-Stat Fourier Decomposition; Wavelet-Stat tool,
+    TC-RMW tool.
+    
+  * METplus line types: Grid-Stat SL1L2, SAL1L2, VL1L2, VAL1L2, CNT, VCNT;
+    Wavelet Stat: ISC, RMW output file.
+    
+* Feature Relative identifies systematic errors associated with a group
+  of case studies.
+
+  * METplus methods: Feature Relative Use Cases.
+     
+* Relationship between two fields: generates a joint PDF between two fields.
+   
+  * METplus methods: Grid-Diag tool.
+    
+* Subseasonal-to-Seasonal Diagnostics compute indices to establish the
+  ability of the model to predict S2S drivers.
+   
+  * METplus methods: S2S Use Cases.
+    
+What is the standard for comparison that provides a reference level of skill (e.g., persistence, climatology, reference model)?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Climatologies or Reference models may be passed into METplus using the
+following configuration options:
+
+* {MET TOOL}_CLIMO_MEAN
+  
+* {MET TOOL}_CLIMO_STDEV
+   
+* {MET TOOL}_CLIMO_CDF
+   
+This can be found in Grid-Stat, Point-Stat, Gen-Ens-Prod, and Ensemble-Stat
+tools.
+
+What is the geographic location of the model data being evaluated? Are there specific areas of interest for the evaluation?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Masking regions are what METplus uses to define verification areas of
+interest. These can be defined prior to running tools using the
+Gen-Vx-Mask tool, or during run-time using the METPLUS_MASK_DICT options.
+
+What domain should be used to evaluate on the model domain, observation domain (if gridded), or some other domain?
```

Comment:
As before, I think this should probably be reworded to "What domain should be used to evaluate on: the model domain..."

Reply:
added colon to both questions.  The one at the top of the page and this one.